agent/04_agent_configure.sh

```diff
@@ -122,21 +122,7 @@ metadata:
   namespace: cluster0
 spec:
   baseDomain: ${BASE_DOMAIN}
-  clusterInstallRef:
-    group: extensions.hive.openshift.io
-    kind: AgentClusterInstall
-    name: test-agent-cluster-install
-    version: v1beta1
   clusterName: ${CLUSTER_NAME}
-  controlPlaneConfig:
-    servingCertificates: {}
-  platform:
```

Comment:
Platform is used in the Hive controller (not sure how this interacts with the MCE controller?): https://github.com/openshift/hive/blob/master/pkg/controller/clusterdeployment/clusterdeployment_controller.go
I think we should probably set it to `none: {}` until further notice.

Reply:
actually no, let's remove it for now.